generator.py

```diff
@@ -84,39 +88,38 @@ def __init__(self, init_gain: float, base_channels: int,
         # Number of output channels
         self.output_channels = base_channels*output_factor
 
-        # Fully-connected layer with number of output channels * 4 * 4 units
+        # Dimension of noise vector
+        self.n_input = n_input
+
+        # bias initialization constant (see PyTorch Conv2d documentation)
+        self.k = tf.math.sqrt(1 / self.n_input)
+
+        # Initial dense layer (implemented as 1x1 convolution) | Number of output channels*16
         # for reshaping into 4x4 feature maps
-        self.dense = Dense(units=self.output_channels * 4 * 4,
-                kernel_initializer=orthogonal(gain=init_gain))
-        self.cbn = ConditionalBatchNormalization(filters=self.output_channels,
```

Comment:
yes. there was a mistake on our side. In their code they only do the linear layer. However, in the paper they specify CBN and relu. But, in this case this would give us double cbn, relu since we start the next block with it as well